src/lamplib/src/genny/tasks/mothra_service.py

```diff
@@ -0,0 +1,51 @@
+from dataclasses import dataclass
+import subprocess
+import tempfile
+from typing import Optional
+import structlog
+import yaml
+
+MOTHRA_REPO = "git@github.com:10gen/mothra.git"
+TEAMS_DIR = "mothra/teams"
+TEAMS_FILES = ["cloud.yaml", "database.yaml", "rnd_dev_prod.yaml", "star.yaml"]
+TEAMS_FILES_PATHS = [f"{TEAMS_DIR}/{file}" for file in TEAMS_FILES]
+SLOG = structlog.get_logger(__name__)
+
+
+@dataclass
+class Team:
+    name: str
+    support_slack_channel_name: Optional[str] = None
+    support_slack_channel_id: Optional[str] = None
+
+    def __init__(self, **kwargs):
+        # Filter out any kwargs that are not in the class
+        filtered_kwargs = {k: v for k, v in kwargs.items() if k in self.__annotations__}
+
+        for k, v in filtered_kwargs.items():
+            setattr(self, k, v)
+
+
+class MothraService:
+
+    def __init__(self):
+        self.team_map = self._load_team_map()
+
+    def get_team(self, team_name: str) -> Optional[Team]:
```

Comment:
[non-blocking comment] Not 100% sure this function is needed (up to you!), but, I want to thank you for not overloading an operator to do this!